### PR TITLE
Turn Proxy - Unify Logging Output

### PR DESCRIPTION
**Pull request by mgherghe**

### CODE REVIEW

File: service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/internal/TurnRestServiceHandler.java [hunk 1]
```diff
@@ -75,6 +78,7 @@ protected void doSessionOpened(HttpAcceptSession session) throws Exception {
             TurnRestCredentials credentials = credentialGenerator.generate(subject);
             username = credentials.getUsername();
             password = credentials.getPassword();
+            LOGGER.info(String.format("Generated username: %s, generated password: %s", username, new String(password)));
```
Comment (mgherghe) [inline service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/internal/TurnRestServiceHandler.java]: Should anything else be logged besides the username and password?

File: transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java [hunk 2]
```diff
@@ -134,14 +135,8 @@ public static boolean addIfNeeded(Logger logger, IoSession session, String trans
         if (!logger.isInfoEnabled()) {
             return false;
         }
-        String user = getUserIdentifier(session);
         String loggingFilterName = transportName + "#logging";
```
Comment (dpwspoon) [inline transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java]: why did you replace this user?

File: service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/internal/TurnRestServiceHandler.java (hunk 1 above)
Comment (dpwspoon) [inline service/turn.rest/src/main/java/org/kaazing/gateway/service/turn/rest/internal/TurnRestServiceHandler.java]: Don't log the password, log the username with the session

File: transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java (hunk 2 above)
Comment (mgherghe) [inline transport/spi/src/main/java/org/kaazing/gateway/transport/LoggingFilter.java]: I extracted the logic for getting the user into the `getLoggingFormat` method in order to be able to also use this in derived classes of the LoggingFilter class.

**Pull request merged**